### PR TITLE
Fix compilation problems by upgrading versions

### PR DESCRIPTION
**Pull request by manolo**
This change is [Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/223)

### CODE REVIEW

**Comment by manolo** [comment]
Ping @tbroyer

**Comment by branflake2267** [comment]
nice job

File: package.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-  "devDependencies": {
-    "grunt": "~0.4.1",
-    "grunt-cli": "~0.1.13",
-    "grunt-contrib-concat": "~0.3.0",
-    "grunt-contrib-uglify": "~0.2.7",
-    "grunt-contrib-imagemin": "~0.4.0",
-    "grunt-contrib-watch": "~0.5.3",
-    "grunt-contrib-less": "~0.11.4"
+  "devDependencies": {},
+  "dependencies": {
```
Comment (tbroyer) [inline package.json]: Any specific reason for moving those from `devDependencies` to `dependencies`?

**Comment by manolo** [comment]
Review status: 0 of 3 files reviewed at latest revision, 1 unresolved discussion.

---

*[package.json, line 5 at r1](https://reviewable.io:443/reviews/gwtproject/gwt-site/223#-Ke3pyJ81Zx92F9kbAkn-r1-5:-Ke9Z4x8R_fAwDw29piM:bhobas) ([raw file](https://github.com/gwtproject/gwt-site/blob/eb28660f2a7485e9f7a03dad72742b353453a2ac/package.json#L5)):*
<details><summary><i>Previously, tbroyer (Thomas Broyer) wrote…</i></summary><blockquote>

Any specific reason for moving those from `devDependencies` to `dependencies`?
</blockquote></details>

No reason, I just removed all dependencies and used `npm --save` to compute latest versions, it updated the dependencies block. I could have used `--save-dev` instead.

Moved to devDependencies now.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/gwtproject/gwt-site/223)*

**Pull request merged**